apis/beacon/deposit_snapshot.yaml

```diff
@@ -0,0 +1,34 @@
+  get:
```

Comment:
should there be support for downloading historical snapshots? ie when checkpoint-syncing, one will want to have a matching snapshot to the state that's being synced, which may or may not (due to race conditions or checkpoint age choice) be the same as the snapshot being served by "default"

Reply:
I think the main idea is just to get a starting point rather than download the whole deposit log, so I think there's probably not a need currently to get historical snapshots...